integration/kubernetes/init.sh

```diff
@@ -155,7 +156,10 @@ echo "Kubelet options:"
 sudo cat /var/lib/kubelet/kubeadm-flags.env
 sudo systemctl daemon-reload && sudo systemctl restart kubelet
 
-kubectl get nodes
+for i in 1 3 7
```

Comment:
ooi, why this set of numbers? Might be clearer to say:

```suggestion
for i in $(seq 3)
```

Reply:
just because i saw the number 10s mentioned somewhere so together it sums (although it might not really needed here)